source/Private/New-GuestConfigurationPolicyContent.ps1

```diff
@@ -71,12 +75,41 @@ function New-GuestConfigurationPolicyContent
         Parameter = $Parameter
     }
 
+    if ($ContentManagedIdentity)
```

Comment:
This is not a boolean.
Please check for not null or not empty or if the parameter exists or if it's the right parameter set.
There are too many things in PS that can make this false without you meaning it.

Reply:
Made the necessary change.